README.dev.md

```diff
@@ -115,16 +115,16 @@ If you really want to install everything by hand, the below
 instructions should work.
 
 Install the following non-Python dependencies:
- * libffi-dev — needed for some Python extensions
- * postgresql 9.1 or later — our database (client, server, headers)
+ * libffi-dev  needed for some Python extensions
+ * postgresql 9.1 or later  our database (client, server, headers)
  * nodejs 0.10 (and npm)
  * memcached (and headers)
  * rabbitmq-server
  * libldap2-dev
  * python-dev
- * redis-server — rate limiting
- * tsearch-extras — better text search
- * libfreetype6-dev — needed before you pip install Pillow to properly generate emoji PNGs
+ * redis-server  rate limiting
+ * tsearch-extras  better text search
+ * libfreetype6-dev  needed before you pip install Pillow to properly generate emoji PNGs
```

Comment:
Oops, seems like a problem with my rebase, I think I messed it up. I'll fix it now and squash.